### PR TITLE
Add email address to addon notification field

### PR DESCRIPTION
**Pull request by tarukumar**
Signed-off-by: Tarun Kumar <takumar@redhat.com>

### CODE REVIEW

**Review by aloganat — APPROVED** [review]
+1, lgtm

File: utils/scripts/ocm/ocm.py [hunk 1]
```diff
@@ -658,6 +658,26 @@ def wait_for_osd_cluster_to_get_deleted(self, timeout=3600):
                    " EXITING".format(self.cluster_name))
             sys.exit(1)
 
+    def update_notification_email_address(self, addon_name, email_address):
+        """Update notification email to Addons"""
+        replace_vars = {
+                       "EMAIL_ADDER": email_address
+                       }
```
Comment (jgarciao) [inline utils/scripts/ocm/ocm.py:665]: Shouldn't it be EMAIL_ADDRE or EMAIL_ADDRESS ?

**Review by jgarciao — APPROVED** [review]
Added a little suggestion but otherwise lgtm

File: utils/scripts/ocm/ocm.py (hunk 1 above)
Comment (tarukumar) [reply utils/scripts/ocm/ocm.py:665]: I just kept it as short intentionally> It is just a simple variable for render

**Pull request merged**